x/genutil/utils.go

```diff
@@ -98,3 +100,80 @@ func InitializeNodeValidatorFilesFromMnemonic(config *cfg.Config, mnemonic strin
 
 	return nodeID, valPubKey, nil
 }
+
+func InitializeNodeValidatorFilesWithKeyType(config *cfg.Config, keyType string) (nodeID string, valPubKey cryptotypes.PubKey, err error) {
```

Comment:
So now `InitializeNodeValidatorFile` should call `InitializeNodeValidatorFilesWithKeyType` with keytype ed25519